package.json

```diff
@@ -10,7 +10,7 @@
     "@ledgerhq/hw-transport-webusb": "^6.27.13",
     "@phosphor-icons/react": "^2.0.10",
     "@react-spring/web": "^9.6.1",
-    "@secretkeylabs/xverse-core": "3.0.0",
+    "@secretkeylabs/xverse-core": "3.1.1-7e91800",
```

Comment:
How does the suffix work? Will we remove it later on for prod?

Reply:
yeah, once we can finalise and publish the next xverse-core version.

the suffixed versions are created from CI on each xverse-core PR